articles/azure/azs-how-create-vm-image-portal.md

```diff
@@ -0,0 +1,121 @@
+---
+title: How to create a custom image from a VM on Azure Stack using the UKCloud Azure Stack portal | UKCloud Ltd
+description: Provides details on how to create a custom image from a VM using the portal
+services: azure-stack
+author: Daniel Brennand
+reviewer:
+lastreviewed:
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a custom image from a VM using the UKCloud Azure Stack portal
+toc_fullpath: Users/How To/azs-how-create-vm-image-portal.md
+toc_mdlink: azs-how-create-vm-image-portal.md
+---
+
+# How to create a custom image from a VM on Azure Stack using the UKCloud Azure Stack portal
+
+## Overview
+
+You can create an image resource from a generalised virtual machine (VM) that is stored as either a managed disk or an unmanaged disk in a storage account. You can then use the image to create multiple VMs.
+
+This article explains how to create a custom image from a VM on Azure Stack, which you can then use to deploy other VMs.
+
+## Prerequisites
+
+To complete the steps in this article, you must have appropriate access to a subscription in the Azure Stack portal.
+
+## [1. Generalise your VM](#tab/tabid-1)
```

Comment:
I don't think this has come out the way you wanted it to... The generalise steps are at the bottom of the page instead of in the "Generalise" section...

Reply:
It appears as expected once the HTML is generated.